build.zig.zon

```diff
@@ -0,0 +1,13 @@
+.{
+    .name = "sqids",
```

Comment:
This declares the package as being named `sqids`.
The package itself expose a module named `sqids` as well, per the declaration in `build.zig`.
Then the user as full liberty to rename it as he wishes.

What is the conventions regarding the naming in other language? Should we go for `sqids-zig` or `sqids` for the package name?

Reply:
`sqids` sounds good to me, unless there's a better convention for Zig. The repo is named that way only because we have other languages.